README.rst

```diff
@@ -425,9 +471,27 @@ Constructors for Sorted Containers
   the parameters of the type.  Ordering argument ``o`` is
   optional and defaults to ``Forward``.
 
+``SortedMultiDict(k,v,o)``
+  Construct a SortedMultiDict using keys given by ``k``, values
+  given by ``v`` and ordering object ``o``.  The ordering object
+  defaults to ``Forward`` if not specified.  The two arguments
+  ``k`` and ``v`` are 1-dimensional arrays of the same length in 
+  which ``k`` holds keys and ``v`` holds the corresponding values.
+  To construct an empty SortedMultiDict with given types ``K`` and
+  ``V``, use
+  ``SortedMultiDict(K[], V[], o)``.
```

Comment:
For Julia 0.4, the standard way to create dictionaries and arrays will be to explicitly supply the types, as 

```julia
julia> Dict{Int, String}()
Dict{Int64,AbstractString} with 0 entries

julia> Array{Int,2}(2,3)
2x3 Array{Int64,2}:
 21474836485  0  0
           1  0  0


julia> Vector{Float64}()
0-element Array{Float64,1}
```